src/AffectedUserFlow/exposureNotificationAPI.ts

```diff
@@ -9,25 +9,42 @@ const defaultHeaders = {
   accept: "application/json",
 }
 
+const EXISTING_KEYS_SENT_RESPONSE = "no revision token, but sent existing keys"
```

Comment:
This one ties directly to the case where:

- No new keys were sent
- No revision token was sent

This happens when there is a new submission on the same device for a different jurisdiction on a very short amount of time(typically testing) There is a different scenario where some keys are inserted that would stop being treated as an error on a future update.